v2/cli/term/event_page.go

```diff
@@ -35,14 +38,32 @@ func newEventPage(
 		eventInfo:  tview.NewTextView().SetDynamicColors(true),
 		workerInfo: tview.NewTextView().SetDynamicColors(true),
 		jobsTable:  tview.NewTable().SetSelectable(true, false),
+		eventLogs:  tview.NewTextView().SetDynamicColors(true),
 		usage: tview.NewTextView().SetDynamicColors(true).SetText(
-			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
+			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](W) [white]Logs    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
 		),
+		logsClient: apiClient.Events().Logs(),
 	}
 	e.eventInfo.SetBorder(true).SetBorderColor(tcell.ColorYellow)
 	e.workerInfo.SetBorder(true).SetBorderColor(tcell.ColorYellow).
 		SetTitle("Worker")
 	e.jobsTable.SetBorder(true).SetTitle("Jobs")
+	e.eventLogs.SetBorder(true).SetTitle("Logs (<-/Del) Quit")
+
+	// Returns a new primitive which puts the provided primitive in the center and
+	// sets its size to the given width and height.
+	modal := func(p tview.Primitive, width, height int) tview.Primitive {
+		return tview.NewFlex().
+			AddItem(nil, 0, 1, false).
+			AddItem(tview.NewFlex().SetDirection(tview.FlexRow).
+				AddItem(nil, 0, 1, false).
+				AddItem(p, height, 1, false).
+				AddItem(nil, 0, 1, false), width, 1, false).
+			AddItem(nil, 0, 1, false)
+	}
+
+	e.logModal = modal(e.eventLogs, 85, 25)
```

Comment:
I don't understand what the rationale might have been for defining a function here and then immediately using that function. Wouldn't this be simpler and more readable/understandable to just do this?

```
e.logModal = tview.NewFlex().
	AddItem(nil, 0, 1, false).
	AddItem(tview.NewFlex().
	SetDirection(tview.FlexRow).
	AddItem(nil, 0, 1, false).
	AddItem(e.eventLogs, 25, 1, false).
	AddItem(nil, 0, 1, false), 85, 1, false).
	AddItem(nil, 0, 1, false)
```